rnd/autogpt_server/autogpt_server/blocks/basic.py

```diff
@@ -137,49 +136,6 @@ def run(self, input_data: Input) -> BlockOutput:
             yield "missing", input_data.input
 
 
-T = TypeVar("T")
-
-
-class InputOutputBlockInput(BlockSchema, Generic[T]):
-    value: T = Field(description="The value to be passed as input/output.")
-    name: str = Field(description="The name of the input/output.")
-
-
-class InputOutputBlockOutput(BlockSchema, Generic[T]):
-    result: T = Field(description="The value passed as input/output.")
-
-
-class InputOutputBlockBase(Block, ABC, Generic[T]):
-    @abstractmethod
```

Comment:
The idea of this base class is allowing strong typing for input output.
Currently there is only Input[Any], but we definitely need Into put Strong input etc later for better UI display and type safety.